gulpfile.js

```diff
@@ -1106,6 +1121,67 @@ function compileJs(srcDir, srcFilename, destDir, options) {
   }
 }
 
+/**
+ * Given a file path `foo/bar.js`, transpiles the TypeScript entry point of
+ * the same name `foo/bar.ts` and all direct and indirect TypeScript imports.
+ *
+ * @param {string} srcDir
+ * @param {string} srcFilename
+ */
+function transpileTs(srcDir, srcFilename) {
```

Comment:
Should this function live in gulpfile? Or be imported from standalone location?

Reply:
Good point, this file is huge. Done.